src/Sulu/Bundle/AudienceTargetingBundle/Entity/TargetGroupConditionRepository.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\AudienceTargetingBundle\Entity;
+
+use Sulu\Component\Persistence\Repository\ORM\EntityRepository;
+
+/**
+ * Repository class for target group conditions.
+ */
+class TargetGroupConditionRepository extends EntityRepository implements TargetGroupConditionRepositoryInterface
```

Comment:
I thought this is making things easier, since i can also typehint repositories when using them.
So i should also remove it from the orm file as well?